### PR TITLE
Flexslider 2.3

### PR DESCRIPTION
**Pull request by smeeckaert**
Here is an update for the slideshow to flexslider 2.3.

There is also 2 little changes : 
- Bugfix with flexpreview which always displayed the next item instead of the current. I took a little time to refactorate using css in order to get a better behaviour with different nav style.
- Hide the select list if we only have one type of slider enabled.

I also update the README to be more usable.

### CODE REVIEW

**Comment by Meroje** [comment]
:+1:

File: config/controller/admin/enhancer.config.php [hunk 1]
```diff
@@ -36,16 +36,23 @@
...
         'view' => 'noviusos_slideshow::admin/enhancer/preview',
     ),
 );
+
+// Don't display list of format if we only have one format
+if (count($formats) > 1) {
+    $config['fields']['format'] = array(
+        'label' => __('Format:'),
+        'form'  => array(
+            'type'    => count($formats) === 1 ? 'hidden' : 'select',
```
Comment (vrcAlbert) [inline config/controller/admin/enhancer.config.php]: Why keeping this? You explicitely add this when "count > 1".
The other question is : if it was hidden, why removing it when count == 1 ? 
(except for avoiding the display of the label, which should be handled indeed).

Comment (smeeckaert) [inline config/controller/admin/enhancer.config.php]: It was a mistake i made while merging projects. It should be corrected now and the enhancer preview will hide formats if there is only one defined.

**Comment by vrcAlbert** [comment]
:+1:
much better :)

**Comment by smeeckaert** [comment]
Splitting the PR into #14 and #15

**Pull request closed without merging**